### PR TITLE
DRIFT-481: Add mkdocs documentation

### PR DESCRIPTION
**Pull request by atimin**

### CODE REVIEW

File: pkg/drift_client/__init__.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 """
 
 from drift_client.drift_client import DriftClient
+from drift_client.package import Package
```
Comment (aschenbecherwespe) [inline pkg/drift_client/__init__.py]: update to new name?

File: pkg/drift_client/package.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+    def dec(self):
+        if self._pkg.status != StatusCode.GOOD:  # pylint: disable=protected-access
+            raise ValueError("Bad package")
+        return func(self)
+
+    return dec
+
+
+class Package:  # pylint: disable=no-member
```
Comment (aschenbecherwespe) [inline pkg/drift_client/package.py]: Name is vague, maybe DriftDataPackage or something would be better.

File: pkg/drift_client/package.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        return self._pkg.publish_timestamp.ToMilliseconds() / self.TS_PRECISION
+
+    @property
+    def status_code(self) -> int:
+        """Status Code
+
+        Returns:
+            Status of the package. Ok (0) means the package is valid
+        """
+        return self._pkg.status
```
Comment (aschenbecherwespe) [inline pkg/drift_client/package.py]: Isn't there some Enum type for this?

File: tests/package_test.py [hunk 4]
```diff
@@ -10,7 +10,7 @@
 from drift_protocol.common import DriftPackage, StatusCode, DataPayload
 from google.protobuf.any_pb2 import Any
 
-from drift_client.drift_client import Package
+from drift_client import Package
```
Comment (aschenbecherwespe) [inline tests/package_test.py]: update to new name, also, should it not be imported from `drift-client.package`?

**Review by aschenbecherwespe — CHANGES_REQUESTED** [review]

File: tests/package_test.py (hunk 4 above)
Comment (atimin) [reply tests/package_test.py]: no, it is a unit test, and we import classes as a user should import them.

File: pkg/drift_client/package.py (hunk 3 above)
Comment (atimin) [reply pkg/drift_client/package.py]: Currently. a user can use protobuf StatusCode enum.

**Review by aschenbecherwespe — APPROVED** [review]

**Pull request merged**